src/simpleTrianglePatterns/simpleTrianglePattern.java

```diff
@@ -3,10 +3,10 @@
 import java.util.Scanner;
 public class simpleTrianglePattern {
     public static void main(String[] args){
-        Scanner scan = new Scanner(System.in);
+        Scanner userInputScanner = new Scanner(System.in);
         System.out.print("Enter a number:");
         try{
-            int n = scan.nextInt();
+            int userInputScanner = scan.nextInt();
```

Comment:
How can be an int a scanner? The int is just a primitive variable type which can store whole numbers (integers) Its not meaningful its misleading.